agent/newrelic-install.sh

```diff
@@ -239,12 +242,21 @@ case "${arch}" in
 esac
 
 # allow override of detected arch
-[ "${NR_INSTALL_ARCH}" = "x86" -o "${NR_INSTALL_ARCH}" = "x64" -o "${NR_INSTALL_ARCH}" = "x86_64" ] && arch="${NR_INSTALL_ARCH}"
+[ "${NR_INSTALL_ARCH}" = "x64" -o "${NR_INSTALL_ARCH}" = "x86_64" ] && arch="${NR_INSTALL_ARCH}"
 
 # exit if arch is unsupported
-if [ "${arch}" != "x86" -a "${arch}" != "x64" ]; then
-  error "An unsupported architecture "${arch}" detected."
+if [ "${arch}" != "x64" ]; then
+    msg=$(
+    cat << EOF
+
+An unsupported architecture detected.
+Please visit:
+  https://docs.newrelic.com/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements/
+to view compatibilty requirements for the the New Relic PHP agent.
```

Comment:
Should we add `The install will now exit.`?
```suggestion
to view compatibilty requirements for the the New Relic PHP agent.
The install will now exit.
```